### PR TITLE
Remove invalid code from core and schema`

### PR DESCRIPTION
**Pull request by wtrocki**
Also unified mapping for CRUD names. It also fixed entire runtime to work with the new model. Not sure what to do with production-based migrations

### CODE REVIEW

File: packages/graphback-core/tsconfig.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "include": [
-    "./src/**/*.ts"
```
Comment (wtrocki) [inline packages/graphback-core/tsconfig.json]: Commited by mistake. To be removed

File: packages/graphback-core/src/crud/mappingHelpers.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { GraphbackOperationType} from '../api';
```
Comment (wtrocki) [inline packages/graphback-core/src/crud/mappingHelpers.ts]: @craicoverflow This is the only change in this PR. I have added one place for mapping every CRUD element. This needs to be extended and used in plugins

**Comment by wtrocki** [comment]
What is outstanding for this is to fix database migration.

**Comment by wtrocki** [comment]
@craicoverflow Looking for feedback on what we do with production migration. 
There is full flow ongoing refactoring for production migration required that could take time. We could remove it for the moment and create task to restore them if needed. WDYT?

File: packages/graphback-core/src/crud/mappingHelpers.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { GraphbackOperationType } from './GraphbackOperationType';
+
+// TODO Talk with Enda. Should mapping be dynamic and swapable or we going strong with CRUD spec
```
Comment (wtrocki) [inline packages/graphback-core/src/crud/mappingHelpers.ts]: So question. Are we going strong with CRUD model (I believe we should to keep things simple or allowing peole to override it if needed and provide custom names. This is very simple change in terms of code but it has tons of side impacts on docs etc.

**Comment by wtrocki** [comment]
@craicoverflow Looking for feedback on what we do with production migration. 
There is full flow ongoing refactoring for production migration required that could take time. We could remove it for the moment and create task to restore them if needed. WDYT?

**Comment by wtrocki** [comment]
Production migration will require day to carefully test it and make sure that it works. InputType context is used there with some extra flags so there will be need for separate vistor etc. I totally forgot about this and thanks for typescript I got nicely reminder in form of 71 compilation errors

**Comment by craicoverflow** [comment]
> @craicoverflow Looking for feedback on what we do with production migration.
> There is full flow ongoing refactoring for production migration required that could take time. We could remove it for the moment and create task to restore them if needed. WDYT?

I think we could remove this for the time being as it is not in use anywhere.

**Pull request merged**